woocommerce/woocommerce
# Bugfix/multiple notices in cart

### All Submissions:


### Changes proposed in this Pull Request:

1. Fixed multiple debug notices in cart:
- first cart view:
![image](https://user-images.githubusercontent.com/8124521/91668189-0c321d80-eb0b-11ea-99ca-d267ea85b729.png)
- after cart update:
![image](https://user-images.githubusercontent.com/8124521/91668192-18b67600-eb0b-11ea-8e56-38eafb781672.png)

2. Notices wrapper not present in pages:
![image](https://user-images.githubusercontent.com/8124521/91668336-4c45d000-eb0c-11ea-9e41-a31bfcd1c590.png)

3. Debug mode not works in checkout

### How to test the changes in this Pull Request:

Multiple notices:
1. Enable debug mode in shipping
2. Add phisical product to cart
3. Go to cart
4. Change quantity
5. Click update cart

Notices wrapper not present in pages:
1. Enable debug mode in shipping
2. Go to shop page
3. Add phisical product to cart
4. Refresh shop page
5. Check page source for `<div class="woocommerce-notices-wrapper">`

Debug mode not works in checkout:
1. Enable debug mode in shipping
2. Add phisical product to cart
3. Go to checkout
4. Change address
5. Debug message should be shown

### Other information:




### Changelog entry

Fixed notices in cart and checkout.
